pkg/kubelet/server/server.go

```diff
@@ -297,8 +324,11 @@ func (s *Server) InstallDefaultHandlers(enableCAdvisorJSONEndpoints bool) {
 		Operation("getPods"))
 	s.restfulCont.Add(ws)
 
+	s.addMetricsBucketMatcher(regexp.MustCompile(`^stats(/.*)?$`), "stats")
 	s.restfulCont.Add(stats.CreateHandlers(statsPath, s.host, s.resourceAnalyzer, enableCAdvisorJSONEndpoints))
+
 	//lint:ignore SA1019 https://github.com/kubernetes/enhancements/issues/1206
+	s.addMetricsBucketMatcher(regexp.MustCompile(`^metrics/?$`), "metrics")
```

Comment:
Needs to go above the lint rule (the lint:ignore only applies to the following line)

Reply:
Fixed.